src/Entity/CommitteeElection.php

```diff
@@ -2,13 +2,30 @@
 
 namespace App\Entity;
 
+use ApiPlatform\Core\Annotation\ApiResource;
 use App\Entity\VotingPlatform\Designation\AbstractElectionEntity;
 use App\Entity\VotingPlatform\Designation\Designation;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Serializer\Annotation\Groups;
 
 /**
+ * @ApiResource(
+ *     routePrefix="/v3",
+ *     itemOperations={
+ *         "get": {
+ *             "path": "/committee_elections/{uuid}",
+ *             "requirements": {"uuid": "%pattern_uuid%"},
+ *             "security": "is_granted('ROLE_OAUTH_SCOPE_JEMENGAGE_ADMIN') and is_granted('IS_FEATURE_GRANTED', 'committee') and is_granted('MANAGE_ZONEABLE_ITEM__FOR_SCOPE', object.getCommittee())",
+ *             "normalization_context": {
+ *                 "groups": {"committee_election:read"},
+ *             },
+ *         },
+ *     }
```

Comment:
could you add `collectionOperations={}` this will disable all operation on collection, otherwise Api-platform open default collection routes like `post`.